var/spack/repos/builtin/packages/gatk/package.py

```diff
@@ -37,19 +37,28 @@ class Gatk(Package):
     url      = "https://software.broadinstitute.org/gatk/download/auth?package=GATK"
     _urlfmt  = "https://github.com/broadgsa/gatk-protected/archive/{0}.tar.gz"
```

Comment:
Can you update the homepage to https://software.broadinstitute.org/gatk/. The old link is dead.

Can you also remove the `_urlfmt` and replace the `url` with the new one? Then we might actually be able to find new versions automatically.